common/src/main/java/tachyon/security/UserInformation.java

```diff
@@ -95,10 +100,13 @@ static String getOsPrincipalClassName() {
         new AppConfigurationEntry(OS_LOGIN_MODULE_NAME,
             AppConfigurationEntry.LoginModuleControlFlag.REQUIRED, BASIC_JAAS_OPTIONS);
 
-    // TODO: define AppConfigurationEntry TACHYON_LOGIN
+    private static final AppConfigurationEntry TACHYON_LOGIN =
+        new AppConfigurationEntry(TachyonLoginModule.class.getName(),
+            AppConfigurationEntry.LoginModuleControlFlag.REQUIRED,
+            BASIC_JAAS_OPTIONS);
```

Comment:
Please double check the indentation---it seems line 105 doesn't need to break before exceeding linewidth limit.